### PR TITLE
[haxe/en] Few cosmetic fixes

### PR DESCRIPTION
**Pull request by sm4rk0**
* correct the array name in a for loop iterating on an index
* add an assignment for "case _" in first switch example
* fix case of variable name in trace for control statement returning as an expression
* move some inline comments to a separate line above the statement and shorten some comments to prevent text overflow

### CODE REVIEW

File: haxe.html.markdown [hunk 1]
```diff
@@ -495,8 +495,10 @@ class LearnHaxe3{
         // foo_instance.public_read = 4; // this will throw an error if uncommented:
         // trace(foo_instance.public_write); // as will this.
 
-        trace(foo_instance + " is the value for foo_instance"); // calls the toString method
-        trace(foo_instance.toString() + " is the value for foo_instance.toString()"); // same thing
+         // calls the toString method
```
Comment (vendethiel) [inline haxe.html.markdown]: please add colon for this comment and the next one

File: haxe.html.markdown [hunk 2]
```diff
@@ -534,9 +536,10 @@ class FooClass extends BarClass implements BarInterface{
 
     // a public constructor
     public function new(arg:Int){
-        super(); // call the constructor of the parent object, since we extended BarClass
+        // call the constructor of the parent object, since we extended BarClass
```
Comment (vendethiel) [inline haxe.html.markdown]: same here

**Comment by vendethiel** [comment]
Looks good outside from that :)

File: haxe.html.markdown (hunk 1 above)
Comment (sm4rk0) [inline haxe.html.markdown]: OK. I actually considered adding a colon, but decided not to do so in order to maintain the same style as the rest of the code, which is mainly without colons.
I hope I'll figure out how to make this changes, as I'm new here...

**Comment by vendethiel** [comment]
Thanks! :cake:

**Pull request merged**